packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -143,6 +143,7 @@ export function SelectBase<T>({
   width,
   isValidNewOption,
   formatOptionLabel,
+  classNamePrefix,
```

Comment:
@lpskdl hm, not sure we should do this... seems a bit too messy and adds yet another new prop to our already massive list of select props 🤔 

wonder if a better solution would be to:
- remove the explicit color set in `SingleValue` here: https://github.com/grafana/grafana/blob/lpskdl/56359/packages/grafana-ui/src/components/Select/SingleValue.tsx#L16 so it inherits the color on `Select`
- then just pass `color` directly to `Select`

it won't change the icon color, but i'm not sure we're too worried about that tbh 🤷